sql-plugin/src/main/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleIterator.scala

```diff
@@ -306,10 +310,12 @@ class RapidsShuffleIterator(
     }
 
     val blockedStart = System.currentTimeMillis()
-    val result = resolvedBatches.take()
+    var result: Option[ShuffleClientResult] = None
+    val timeoutSeconds = rapidsConf.shuffleFetchTimeout
```

Comment:
might make this a class variable so you don't have to keep re-reading unless you think its changing dynamically

Reply:
Updated @tgravescs